blazor/rich-text-editor/style.md

```diff
@@ -61,21 +107,87 @@ Use the following CSS to customize the default color in the Rich Text Editor's t
     color: red;
 }
 ```
+![Blazor RichTextEditor with customizing editor toolbar](./images/blazor-richtexteditor-editor-toolbar.png)
 
-## Customizing the Rich Text Editor's character count
+### Refresh editor
 
-Use the following CSS to customize the default color in the Rich Text Editor's character count.
+While rendering the Rich Text Editor inside the dialog component, the dialog container and its wrapper elements are styled with display: none, so the editor’s toolbar does not get the proper offset width and will render above the edit area container. To resolve this issue, call the `RefreshUI` method of the RichTextEditor in the dialog's opened event.
 
-```css
-/* To change font color, font family, font size and opacity  */
-.e-richtexteditor .e-rte-character-count {
-    color: red;
-    font-family: segoe ui;
-    font-size: 18px;
-    opacity: 00.54;
-    padding-bottom: 2px;
-    padding-right: 14px;
+{% tabs %}
+{% highlight razor %}
+
+{% include_relative code-snippet/refresh.razor %}
+
+{% endhighlight %}
+{% endtabs %}
+
+![Blazor RichTextEditor with refresh editor](./images/blazor-richtexteditor-refresh-editor.png)
+
+## Highlight the specific lines
+
+Programmatically highlight a portion of the text in the editor, like setting the background color of the text, by applying background style to the particular text using the RichTextEditor [ExecuteCommand](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.RichTextEditor.ExecuteCommandOption.html) method and the `jsinterop` method.
```

Comment:
Programmatically highlight a portion of the text in the editor, like setting the background color of the text by applying background style to the particular text using the RichTextEditor [ExecuteCommand](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.RichTextEditor.ExecuteCommandOption.html) method and the `jsinterop` method.